obal/data/roles/spec_file/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
```

Comment:
I think this also depends on `package_variables` to define `package_base_dir`.

Reply:
`ensure_package` depends on `package_variables` so I didn't feel the need to specify it here also.